rmgpy/data/kinetics/library.py

```diff
@@ -141,7 +141,7 @@ def getLibraryReactions(self):
         rxns = []
         for entry in self.entries.values():                
             if entry._longDesc and 'Originally from reaction library: ' in entry._longDesc:
-                lib = entry._longDesc.replace('Originally from reaction library: ','')
+                lib = entry._longDesc.split('\n')[0].replace('Originally from reaction library: ','')
```

Comment:
Does this have the same reliability concerns as the [first commit](https://github.com/ReactionMechanismGenerator/RMG-Py/pull/1200/commits/d81e5d6dd7772b3b688a73977a2ef9b35d51e75f)

Reply:
I've changed it now so that it extracts the library name from the first line that has 'Originally from reaction library:  ' in it